src/test.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH="${PYTHONPATH}:$HOME/3rdparty/TheanoNetCore/TheanoNetCoresrc"
```

Comment:
this should not be needed as tnetcore is already contained inside this repository